java/examples/pom.xml

```diff
@@ -50,6 +50,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
+      <scope>compile</scope>
```

Comment:
We need to fix line 57 for `hadoop-hdfs`, too. Could you fix it together?

Reply:
OK